packages/markdown/remark/src/index.ts

```diff
@@ -20,14 +25,25 @@ import rehypeRaw from 'rehype-raw';
 import rehypeStringify from 'rehype-stringify';
 import markdown from 'remark-parse';
 import markdownToHtml from 'remark-rehype';
+import remarkGfm from 'remark-gfm';
 import { unified } from 'unified';
 import { VFile } from 'vfile';
 
 export { rehypeHeadingIds } from './rehype-collect-headings.js';
 export * from './types.js';
 
-export const DEFAULT_REMARK_PLUGINS = ['remark-gfm', 'remark-smartypants'];
-export const DEFAULT_REHYPE_PLUGINS = [];
+export const markdownConfigDefaults: Omit<Required<AstroMarkdownOptions>, 'drafts'> = {
```

Comment:
Move `markdownConfigDefaults` to a `const` so Remark, Astro Core, and MDX can all share the same fallbacks!